### DISCUSSION TITLE
前几天购买2年版账号后通过邮箱登录一直没发送激活码

### DISCUSSION THREAD

**Discussion by nantianhaike** [getlantern/lantern-forum-cn#740] [General]
前几天通过支付宝购买了2年版的lantern账号，一开始使用最新版的windows 稳定版登录，软件一打开直接闪退。后来下载了最新的测试版，操作界面类似App，但是通过邮箱升级到专业版本的时候，点击提交始终没有反应，邮箱也收不到任何激活码

**Comment by nantianhaike** [comment]
算了，还是不要使用最新的版本的lantern，我现在换回windows7版本的蓝灯，可以正常使用了

**Reply by MaxMKMax** [reply]
@nantianhaike 
好的. 谢谢您. 请您等待下一个更好的新版本.
